doc/ref_model/chapters/chapter07.md

```diff
@@ -225,259 +224,57 @@ The platform supports the workload, and in effect controls access to the workloa
+* All API access must use TLS.
+* All production workloads must be separated from all non-production workloads including separation between non-hosted non-production external networks.
 * Where there are multiple hosting facilities used in provision of the service, network communications between facilities for the purpose of backup, management and application communication are cryptographically protected in transit between data centre facilities.
-* Continuous Cloud security compliance is mandatory
-* All data persisted to primary, replica or backup storage is to be encrypted
-* All platform security logs are to be time synchronised
-* Logs are to be regularly scanned for events of interest
-* An incident response plan must exist for the platform
-* the cloud services must be regulalry vulnerability and penetration tested
-* 
-*              --- review and continue this list ---
-*
-
+* Continuous cloud security compliance is mandatory.
+* All data persisted to primary, replica or backup storage is to be encrypted.
+* All platform security logs are to be time synchronised.
+* Logs are to be regularly scanned for events of interest.
+* An incident response plan must exist for the platform.
+* The cloud services must be regularly vulnerability and penetration tested.
 
 <a name="7.5.2"></a>
-## 7.5.2 Platform ‘back-end’ access security
-
-    *(Security requirements around how the platform systems must
-    interconnect with supporting infrastructure services including
-    assurance, fault, asset systems, billing systems, capacity,
-    configuration, and etc.)*
+### 7.5.2 Platform ‘back-end’ access security
+* Restrict traffic to only traffic that is necessary, and deny all other traffic.
+* Use cryptographically-protected protocols for administrative access to the platform.
+* Data Centre Operations staff and systems must use management protocols that limit security risk such as SNMPv3, SSH v2, ICMP, NTP, syslog, and TLS.
+* A platform change management process that is documented, well communicated to staff and tenants, and rigourously followed.
+* A process to check change management adherence that is implemented, and rigourously followed.
+* Processes for managing platform access control filters that are documented, followed, and monitored.
+* No login to root on any platform systems.
+* RBAC must apply for all systems access.
+* An approved system or process for last resort access must exist for the platform.
+* All back-end API access must use TLS.
 
 <a name="7.5.3"></a>
-## 7.5.3 Platform ‘front-end’ access security
-
-    *(Security requirements around how the platform will support network
-    connections that can be used by workloads. Generally the platform will
-    provide the basic connectivity such as a physical MPLS connection, or
-    Internet connection, but the workloads will have a VLAN on that physical
-    connection and provide additional security controls).*
-
-<a name="7.5.4"></a>
-## 7.5.4 Platform services
```

Comment:
Is it so that these sections that are removed are being moved to the RA document?
Is that part of a different PR?

Reply:
@glimonta correct. Chapter 9 I believe in RA1. Please resolve if you are happy.